src/core/storage/query_engine/execution/execution_node.cpp

```diff
@@ -9,23 +9,10 @@
 #include <core/storage/query_engine/execution/query_context.hpp>
 #include <core/storage/query_engine/execution/execution_node.hpp>
 #include <core/system/cppipc/cppipc.hpp>
+#include <util/coro.hpp>
```

Comment:
We should probably put this in core/system/util/.  

To move that file easily, just `git mv` this there, then run `./scripts/repo.py --fix-header coro.hpp` and all the include paths will get corrected.